Makefile

```
@@ -112,6 +115,19 @@ run: go.build
 
 .PHONY: cobertura reviewable manifests submodules fallthrough test-integration run crds.clean
 
+service: 
+	@if [ "$(ServiceID)" = "" ]; then \
+		echo "Error: Please specify value of 'ServiceID'."; \
+		echo "For more info: https://github.com/crossplane/provider-aws/blob/master/CODE_GENERATION.md#code-generation"; \
+		exit 1; \
+	fi
+	@if [ ! -d "$(WORK_DIR)/code-generator" ]; then \
+		cd $(WORK_DIR) && git clone "https://github.com/aws-controllers-k8s/code-generator.git"; \
+		cd $(WORK_DIR)/code-generator && git checkout $(CODE_GENERATOR_COMMIT); \
+	fi
```

```suggestion
		cd $(WORK_DIR) && git clone "https://github.com/aws-controllers-k8s/code-generator.git"
	fi
	cd $(WORK_DIR)/code-generator && git fetch origin && git checkout $(CODE_GENERATOR_COMMIT)
```

It doesn't refresh what's checked out after the first time it runs. I'd suggest adding this change so that it makes sure the requested commit is checked out.

---

@muvaf I have made the changes as you suggested. Please check the commit